pkg/restic/commands.go

```diff
@@ -71,7 +71,8 @@ type restoreParams struct {
 
 func (w *ResticWrapper) listSnapshots(snapshotIDs []string) ([]Snapshot, error) {
 	result := make([]Snapshot, 0)
-	args := w.appendCacheDirFlag([]interface{}{"snapshots", "--json", "--quiet", "--no-lock"})
+	args := w.appendCacheDirFlag([]interface{}{"snapshots", "--json", "--quiet"})
```

Comment:
It is better to use the `--no-lock` flag explicitly instead of going through another function. That will give clear idea which commands are using the flag and which are not.